src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs

```diff
@@ -2,27 +2,24 @@
 // Licensed under the MIT license.
 
 using System.IO;
-using System.Threading.Tasks;
-using Microsoft.CodeAnalysis;
 using Microsoft.PowerFx.Core.Tests;
 using Microsoft.PowerFx.Interpreter.Tests.XUnitExtensions;
-using Microsoft.PowerFx.Types;
 using Xunit;
 using static Microsoft.PowerFx.Interpreter.Tests.ExpressionEvaluationTests;
 
 namespace Microsoft.PowerFx.Interpreter.Tests
-{    
+{
     public class FileExpressionEvaluationTests : PowerFxTest
-    {        
+    {
         [InterpreterTheory]
-        [TxtFileData("ExpressionTestCases", "InterpreterExpressionTestCases", nameof(InterpreterRunner))]        
+        [TxtFileData("ExpressionTestCases", "InterpreterExpressionTestCases", nameof(InterpreterRunner), "en-US, fr-FR")]        
```

Comment:
>"en-US, fr-FR" [](http://example.com/codeflow?start=105&length=14)

Let's discuss.

Reply:
for now, i'll keep "en-US" and we can expand later